test/server.js

```diff
@@ -41,7 +41,9 @@ describe("Server", () => {
 		});
 	});
 
-	describe("WebSockets", () => {
+	describe("WebSockets", function() {
```

Comment:
This is related to https://mochajs.org/#arrow-functions.
It should be fixed across the repo, but it will make sense to do this once https://github.com/lo1tuma/eslint-plugin-mocha/issues/141 gets implemented (and we use that plugin).